awswrangler/s3/_write_parquet.py

```diff
@@ -510,6 +540,15 @@ def to_parquet(  # pylint: disable=too-many-arguments,too-many-locals
         catalog_table_input = catalog._get_table_input(  # pylint: disable=protected-access
             database=database, table=table, boto3_session=session, catalog_id=catalog_id
         )
+        if catalog_table_input:
+            table_type = catalog_table_input["TableType"]
```

Comment:
It was pointed to me that there is no need to pass `table_type="GOVERNED"` if the table already exists and should instead be retrieved from the catalog so I have fixed it here